packages/fuels-types/src/lib.rs

```diff
@@ -31,3 +32,27 @@ pub struct Property {
     pub type_field: String,
     pub components: Option<Vec<Property>>, // Used for custom types
 }
+
+// Both those constants are used to determine if a type field represents an `Enum` or a `Struct`.
+// Since it would have the format `struct foo` or `enum bar`, there is a whitespace.
+pub const STRUCT_KEYWORD: &str = "struct ";
+pub const ENUM_KEYWORD: &str = "enum ";
```

Comment:
Do these need to be `pub`? Would `pub(crate)` suffice?

Reply:
you're right, some leftover from copy-paste